src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public static String getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
+        return String.valueOf(Color.values()[index]);
+    }
+
```

Comment:
remove empty line

Reply:
Done, but why Maven didn't mark this as a mistake when cleaning the package?

Reply:
Done? But I see this line